<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorDialog</class>
 <widget class="QDialog" name="GeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор паролей</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="password_output">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="generate_btn">
       <property name="text">
        <string>Сгенерировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy_btn">
       <property name="text">
        <string>Копировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="length_label">
     <property name="text">
      <string>Длина пароля: 12</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="length_slider">
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="uppercase_check">
     <property name="text">
      <string>Заглавные буквы (A-Z)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="lowercase_check">
     <property name="text">
      <string>Строчные буквы (a-z)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="digits_check">
     <property name="text">
      <string>Цифры (0-9)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="symbols_check">
     <property name="text">
      <string>Символы (!@#$%)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="strength_label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>